src/lib/double_marlin/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
should this VSCode file be in the repo?

Reply:
The file got merged by mistake, will remove.